container-installs/docker-install.sh

```diff
@@ -126,7 +118,7 @@ else
   sudo apt install -y docker-compose-plugin
 fi
 
-# docker instal via snap
+# docker install via snap
```

Comment:
I mustve missed this somewhere. Why are you installing docker with snap?
No need when we can just add the docker repo to apt and install with apt. Which it does seem you're doing also?

You're gonna have a bad time using docker with snap. esp when snap runs its default scheduled updates and stops & restarts snapd, but doesn't restart other processes (in this case docker)... ask me how I know.

Reply:
I was having trouble with apt, so maybe I am not adding docker to the repo correctly. The snap was a prompt from the error msg I kept getting. 

`fortis@trifecta:~$ docker --version Command 'docker' not found, but can be installed with: snap install docker         # version 27.2.0, or apt  install docker.io      # version 24.0.7-0ubuntu4.1 apt  install podman-docker  # version 4.9.3+ds1-1ubuntu0.2 See 'snap info docker' for additional versions.`

Reply:
Get back to the state where you get that error. Then try `sudo docker ...` see if that changes the output

Reply:
I changed the placement of the docker install (to apt), and placed it just after were i add the repository, and it worked fine
<img width="529" alt="Screenshot 2024-12-17 at 8 30 48 PM" src="https://github.com/user-attachments/assets/53f31ed3-f5b1-476b-9f27-edf9c8f15bb1" />

Reply:
OHHHH nvrmnd I see, I had it right I just should have run `sudo docker --version`..... Is that because it installs at the root?

Reply:
I did not need to do anything else, but sudo it. I commented out what I changed then re-pushed, It tested on my end. just give me a approval if that is right?